provision/service.go

```diff
@@ -34,6 +34,13 @@ var (
 	ErrFailedBootstrap          = errors.New("failed to create bootstrap config")
 	ErrFailedBootstrapValidate  = errors.New("failed to validate bootstrap config creation")
 	ErrGatewayUpdate            = errors.New("failed to updated gateway metadata")
+
+	// Limit query parameter
+	Limit uint = 10
```

Comment:
this doesnt need to be exported, right?

Reply:
I can add it without defining it when calling the users function in the SDK i.e 
`if _, err := ps.sdk.Users(token, 0, 10, ""); err != nil {}`

Reply:
Done